sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/thrift/ThriftCLIService.java

```diff
@@ -282,8 +282,8 @@ private String getIpAddress() {
     String clientIpAddress;
     // Http transport mode.
     // We set the thread local ip address, in ThriftHttpServlet.
-    if (cliService.getHiveConf().getVar(
-        ConfVars.HIVE_SERVER2_TRANSPORT_MODE).equalsIgnoreCase("http")) {
+    if ("http".equalsIgnoreCase(cliService.getHiveConf().getVar(
```

Comment:
This file copy from `hive`, It's best not to touch it to avoid conflict

Reply:
OK. If these are copied from Apache Hive, it's better to submit PR to Hive and cherry pick